.github/workflows/rust.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v18
+    - name: Environment
+      run: nix develop .#aiken
```

Comment:
That's not really what I meant 😅 ... 
We need an automated workflow that regularly (i.e. everytime something is merged into the `main` branch) check the repository and propose a PR that updates the Cargo.nix file. 
We could force it on standard CI run and _fail_ if the Cargo.nix is out to date but that's really going to be annoying an "in the way". I don't want to be bothered by Nix stuff when I am not using Nix. So an automated PR that comes to fix a Cargo.nix sounds like a good middleground.